app/src/peripherals/lorawan_handler/lorawan_handler.cpp

```diff
@@ -10,7 +10,10 @@
 
 LOG_MODULE_REGISTER(lorawan_handler, LOG_LEVEL_DBG);
 
-LoRaWANHandler::LoRaWANHandler() {
+Sensor<buzzverse_v1_BQ27441Data>* LoRaWANHandler::battery_sensor = nullptr;
+
+LoRaWANHandler::LoRaWANHandler(Sensor<buzzverse_v1_BQ27441Data>& battery_sensor) {
```

Comment:
It's better to use initializer list:

```suggestion
LoRaWANHandler::LoRaWANHandler(Sensor<buzzverse_v1_BQ27441Data>& sensor)
  : battery_sensor(&sensor)
```